libs/vue-plugin/src/schematics/application/files/src/app/app.vue.template

```diff
@@ -110,4 +131,5 @@ li {
 a {
   color: #42b983;
 }
+<% } %>
```

Comment:
nit: If you move this to the line above, then the code would be formatted more nicely for users who skip formatting. Based on the placement of your other conditionals it looks like you pay attention to this.